### PR TITLE
added survey boolean logic server side

### PR DESCRIPTION
**Pull request by mrkingmarc01**

### CODE REVIEW

**Comment by mrkingmarc01** [comment]
let me know if **>100** and **%20** are fine or other values are better

File: code4me-server/src/api.py [hunk 1]
```diff
@@ -50,10 +51,15 @@ def autocomplete():
             "inferenceTime": (t_after - t_before).total_seconds() * 1000
         }))
 
+    survey = False
+    n_suggestions = len(glob.glob(f"./../data/{user_token}*.json"))
```
Comment (FrankHeijden) [inline code4me-server/src/api.py:55]: Globglogabgalab why the `./`

Comment (mrkingmarc01) [reply code4me-server/src/api.py]: 👍

File: code4me-server/src/api.py [hunk 2]
```diff
@@ -50,10 +51,15 @@ def autocomplete():
             "inferenceTime": (t_after - t_before).total_seconds() * 1000
         }))
 
+    survey = False
+    n_suggestions = len(glob.glob(f"data/{user_token}*.json"))
+    if n_suggestions >= 100 and n_suggestions % 20 == 0:
+        survey = True
```
Comment (FrankHeijden) [inline code4me-server/src/api.py:57]: why not
```suggestion
    n_suggestions = len(glob.glob(f"data/{user_token}*.json"))
    survey = n_suggestions >= 100 and n_suggestions % 20 == 0
```

Comment (mrkingmarc01) [reply code4me-server/src/api.py]: 👍

**Review by FrankHeijden — APPROVED** [review]

**Pull request merged**